.github/workflows/reviewdog.yml

```diff
@@ -1,8 +1,90 @@
-name: reviewdog
-on: [pull_request]
+---
+# Runs various ReviewDog based checks against PR with suggested changes to improve quality
+name: Review
+on:
+  pull_request:
+env:
+  DO_NOT_TRACK: 1
 jobs:
+  prep-review:
+    name: Prepare Review Jobs
+    runs-on: ubuntu-latest
+    outputs:
+      actionlint: ${{ steps.actionlint.outputs.run }}
+      golangci-lint: ${{ steps.golangci-lint.outputs.run }}
+      hadolint: ${{ steps.hadolint.outputs.run }}
+      shellcheck: ${{ steps.shellcheck.outputs.run }}
+      yamllint: ${{ steps.yamllint.outputs.run }}
+    steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          fetch-depth: 0
```

Comment:
not needed

Reply:
Just the fetch-depth part, or both that and the submodules part as well?

Reply:
Both. This repo has no submodules.